lib/model/message_list.dart

```diff
@@ -86,6 +87,30 @@ class MessageListView extends ChangeNotifier {
     notifyListeners();
   }
 
+  Message applyChangesToMessage(UpdateMessageEvent event, Message oldMessage) {
+    Map<String, dynamic> oldMessageJson = oldMessage.toJson();
+
+    //TODO: What other fields should be edited by the update event?
```

Comment:
We'll want to handle all the changes that can happen as part of the message's contents being changed, as described in the API docs:
https://zulip.com/api/get-events#update_message
(but not those that only apply when the message was moved to a different topic and/or stream).

Reply:
Aha! Thanks, I was looking for exactly this doc. Will do.